{
    "type": "bundle",
    "id": "bundle--c176f46f-0b53-407a-a970-633c1773e0a5",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--dfd02a3a-47b3-4629-9eb1-6412fbfaa863",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2021-03-13T20:11:53.673381Z",
            "modified": "2021-03-13T20:11:53.673381Z",
            "name": "CVE-2019-9152",
            "description": "An issue was discovered in the HDF HDF5 1.10.4 library. There is an out of bounds read in the function H5MM_xstrdup in H5MM.c when called from H5O_dtype_decode_helper in H5Odtype.c.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2019-9152"
                }
            ],
            "object_marking_refs": [
                "marking-definition--62fd3f9b-15f3-4ebc-802c-91fce9536bcf"
            ]
        }
    ]
}